09-features.md

```diff
@@ -26,6 +26,7 @@ These flags may only be used in the `init` message:
 | 3  | `initial_routing_sync` | Indicates that the sending node needs a complete routing information dump | [BOLT #7](07-routing-gossip.md#initial-sync) |
 | 4/5  | `option_upfront_shutdown_script` | Commits to a shutdown scriptpubkey when opening channel | [BOLT #2](02-peer-protocol.md#the-open_channel-message) |
 | 6/7  | `gossip_queries`           | More sophisticated gossip control | [BOLT #7](07-routing-gossip.md#query-messages) |
+| 48/49| `option_static_remotekey`   | Static key for remote output | [BOLT #3](03-transactions.md) |
```

Comment:
IMO this should also be a global feature bit, so nodes can opt to _only_ open channels to other nodes that support this new "safu commitment".

Reply:
Also why not in order? So 10/11?

Reply:
Because local features will also be advertized when we stop bikeshedding #571...

The real bit will be assigned when we finally ack this and are doing interop...